canonical/ubuntu.com
# Nvidia partners page

## Done

Build Nvidia partners page

## QA

- Check https://ubuntu-com-12441.demos.haus/nvidia against [copy docs](https://docs.google.com/document/d/1DJ1FuEswAojPNSX9JCLcWJwy-va5VMzuuLGwSNM0lRY/edit#)
- Check https://ubuntu-com-12441.demos.haus/nvidia against [design ](https://www.figma.com/file/S6KvNUt0agl0bcbp6wjhnB/Partners?node-id=576%3A9683&t=mHdjBYcAY7pSpuxX-0)
- Check contact-us form works
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes


## Issue / Card

Fixes https://github.com/canonical/web-squad/issues/6062


## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)
